docs/rule/modifier-name-case.md

```diff
@@ -1,6 +1,6 @@
 # modifier-name-case
 
-:dress: The `extends: 'stylistic'` property in a configuration file enables this rule.
+:nail_care: The `extends: 'stylistic'` property in a configuration file enables this rule.
```

Comment:
These were fixed on next directly a minute or two ago, I think you should be able to rebase and drop these doc changes

Reply:
@rwjblue done